packages/fxa-settings/src/components/en.ftl

```diff
@@ -20,10 +20,14 @@
 # “Account” can be localized, “Firefox” must be treated as a brand.
 # This is used to refer to a user's account, e.g. "update your Firefox account ..."
 -product-firefox-account = Firefox account
-product-mozilla-vpn = Mozilla VPN
-product-pocket = Pocket
-product-firefox-monitor = Firefox Monitor
-product-firefox-relay = Firefox Relay
+-product-mozilla-vpn = Mozilla VPN
+-product-mozilla-mdn = Mozilla MDN
+-product-mozilla-hubs = Mozilla Hubs
+-product-pocket = Pocket
+-product-firefox-monitor = Firefox Monitor
+-product-firefox-relay = Firefox Relay
+-product-firefox-sync = Firefox Sync
```

Comment:
Firefox Sync has been explicitly deprecated, I don't think it should be in the menu (definitely shouldn't be defined as a term).

The guidance was to consider "sync" as an abbreviation for "synchronize/synchronization".